### ISSUE TITLE
Precipitation total only shows for rain, and not snow

### ISSUE THREAD

**Issue by SlidingHorn** [chubin/wttr.in#255]
Title explains...for an example, see the entry for Cleveland for Dec 31 & Jan 1...

[Screenshot](https://i.imgur.com/tlCPTbn.png)

**Comment by RichardDooling** [comment]
I experience the same. I see no data for snow accumulation.

**Comment by webarnes** [comment]
It looks like a limitation of the WorldWeatherOnline source. The field is labelled "Rain" on their website. It's not the customary "Probability of precipitation" figure. In my town, I have the odd situation of having it forecast "Blizzard" with what looks like 0% POP.

I don't think the source handles snow very well. I pulled up historical data for a day last week that I know for a fact (because I shovelled it) had more than 30cm of snow. WorldWeatherOnline totalled 2mm precipitation for the day.

**Comment by chubin** [comment]
Yes, that's true. WWO has many limitations and some other drawbacks, so we should support several backends anyway.

**Comment by webarnes** [comment]
Darksky.net is interesting. It's free for 1000 API calls per day. It has rain and snow forecasts. I've used the app and it sometimes very accurately gives forecasts like "Rain stopping in 11 minutes".

**Comment by chubin** [comment]
Yes, I know about this service, I even experimented with it already. I think that we could at its support
